index.html

```diff
@@ -219,12 +219,136 @@ <h2 class="destinations__title">The best destinations</h2>
     <!--SignUp-->
     <section class="sign-up">
       <div class="sign-up__wrap">
-      <img src="./img/user.svg" alt="user" class="sign-up__image" />
-      <p class="sign-up__title">Sign up and get 20% off your first booking</p>
-      <button class="sign-up__btn">Sign up</button>
-      <button class="sign-up__btn-close">
-        <img src="./img/close.svg" alt="X" class="sign-up__image-close" />
-      </button>
+        <img src="./img/user.svg" alt="user" class="sign-up__image" />
+        <p class="sign-up__title">Sign up and get 20% off your first booking</p>
+        <button class="sign-up__btn">Sign up</button>
+        <button class="sign-up__btn-close">
+          <img src="./img/close.svg" alt="X" class="sign-up__image-close" />
```

Comment:
Text inside alt should be readable for screen readers
Here it's better to use "close" instead of "x"